csdc-6/adcs-simulation/cpp/interface/inc/sim_interface.hpp

```diff
@@ -118,18 +118,20 @@ class Sensor : public ADCS_device {
         virtual ~Sensor(){}
 
         /**
+         * @note commenting out for now but this may need to be removed entirely. Keeping until a decision is made.
+         * 
          * @name    take_measurement
          *
          * @details this function takes a measurement using the sensor. The simulation is also told
          *          to update.
          *
          * @returns the required measurement if succesful.
         **/
-        virtual measurement take_measurement() // MAY need to create a default implementation
-        {
-            measurement empty;
-            return empty;
-        } 
+        // virtual measurement take_measurement() // MAY need to create a default implementation
```

Comment:
again, are we commenting this out or removing it?

Reply:
Leaving this in for future reference